.github/workflows/test.yml

```diff
@@ -0,0 +1,15 @@
+name: CI Angular app through Github Actions
+on: push
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+
+      - name: Frontend Test
+        run: |
+          cd frontend && npm ci && npm test -- --no-watch --no-progress --browsers=ChromeHeadlessCI
```

Comment:
this can't work, right? we are currently sitting in the frontend directory. there is no openproject core linked. just from the frontend, we can't run the tests (as they need to run coming from the core).

Reply:
ja this is not working at all, we'll need to do the same flow as the docker file, but i haven't figure out how to do that in the workflow way yet